packages/lab/src/stepped-tracker/Step.Icon.tsx

```diff
@@ -0,0 +1,69 @@
+import { makePrefixer, useIcon } from "@salt-ds/core";
+import type { IconProps } from "@salt-ds/icons";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import clsx from "clsx";
+import { useMemo } from "react";
+
+import type { Step } from "./Step";
+import stepIconCSS from "./Step.Icon.css";
+
+export namespace StepIcon {
+  export interface Props extends IconProps {
+    stage: Step.Stage;
+    status?: Step.Status;
+    multiplier?: IconProps["size"];
```

Comment:
If this is intentional change, at least make it `sizeMultiplier`

Reply:
Done!